runtests.sh

```diff
@@ -839,7 +841,7 @@ fi
 
 
 echo 'Initializing...'
-initialize > /dev/null 2>&1
+initialize #> /dev/null 2>&1
```

Comment:
Don't pollute the stdout.

Reply:
@debodirno - sure, pls have a look of my previous comment - where I wanted to see what is the error when trying to run the tests using github actions. To see error messages I had to comment the redirection. I a happy to comment it back, but I assume you are not merging without green tests? Let me know pls.

Reply:
I am checking what we can do to get the repository active again for development and contribution.